src/applications/personalization/profile/hooks/useNotificationSettingsUtils.js

```diff
@@ -176,6 +206,26 @@ export const useNotificationSettingsUtils = () => {
           channelId => channels[channelId],
         );
 
+        const isItem3 = itemId === 'item3';
+        const isItem4 = itemId === 'item4';
+        if (
+          isItem3 &&
+          profileShowMhvNotificationSettingsEmailAppointmentReminders &&
+          !getEmailAddress
+        ) {
+          acc.push(item);
+        }
+        if (isItem3 && !getMobilePhone) acc.push(item);
```

Comment:
should these be `else if` statements, just in case the an `item` satisfies both conditionals and is added to `acc` twice? or is there a guarantee this won't occur?

Reply:
🤔 another way to slim this might be to do something like 
```
if (isItem3) {
  const noEmail = profileShowMhvNotificationSettingsEmailAppointmentReminders &&
          !getEmailAddress;
  const noPhone = !getMobilePhone
  if (noEmail || noPhone)
  {
    acc.push(item);
  }
} else if (isItem4) {
  // ...
}
```
not sure that's much better tho...  business logic conditionals are sometimes just a little messy no matter what lol.

Reply:
I liked this approach a lot better. I also added an additional condition as a safety measure @DanielMiller-agile6